crates/bevy_render/src/settings.rs

```diff
@@ -47,6 +47,8 @@ impl Default for WgpuSettings {
     fn default() -> Self {
         let default_backends = if cfg!(feature = "webgl") {
             Backends::GL
+        } else if cfg!(windows) {
```

Comment:
Would this do the trick?

Reply:
```suggestion
        } else if cfg!(target_os = "windows") {
```

Might need to do this, according to https://doc.rust-lang.org/reference/conditional-compilation.html#target_os

Reply:
In this example it shows using the shorter version
https://doc.rust-lang.org/reference/conditional-compilation.html#the-cfg-macro

Reply:
I'd rather us be explicit and consistent here. I think we consistently use `target_os = "blah"` elsewhere in the codebase.